jpteam/Prebid.js
# [Internal Code Review] C1X adapter for Prebid v1.0

Adapter Code is done and tested with my test page. Please **Close** this PR once the code review is done. The adapter branch now has Prebid 1.0 codes but I can only send our changed files with Prebid master as a base. Will sort things out and send another PR to Prebid.org.

A few questions here:
1. siteId/ pixelId/ endpoint settings.
> All adapters must support the creation of multiple concurrent instances. This means, for example, that adapters cannot rely on mutable global variables. - from Prebid website

I am not sure if it is ok to use our own tag under window object. I implemented window.c1x_pubtag in order to follow the internal requirement which is avoid redundancy of bid params in ad units. Ex: the same site id has to be filled in each ad units on a webpage.

Just found out this PR discussion... https://github.com/prebid/Prebid.js/pull/1664
I might need to update our adapter to meet Prebid v1.0 requirements soon.

2. cpm bid price from our bidder - Net or Gross?
To confirm _netRevenue_ in bid response object

3. window obj not exists in automated testing... I didn't find a good way after a full day of research. Any suggestions?
Finding: 
- the unit tests are run in headless chrome by default. Headless chrome doesn't actually has a window or browser viewport. But our common params are put as a key of window object...


# Test Steps

1. $ gulp build --modules=c1xBidAdapter
2.  confirm the build file location on the test page
pbs.src = '../../build/dist/prebid.js';
3. open test page, you should be able to see one ad showing up on the page
